### ISSUE TITLE
Is there a way to create invisible connections between nodes to help with orientation of unconnected nodes?

### ISSUE THREAD

**Issue by jsolbrig** [mermaid-js/mermaid#3192]
I'd like to add a legend to a flowchart that I've been putting together. I'd prefer if there were no connections between the nodes in the legend.

### What I have now
```text
flowchart LR
  subgraph Legend
    direction LR
    stor1[("Storage1")]
    stor2[("Storage2")]
    other["Other Thingy"]
  end
```
```mermaid
flowchart LR
  subgraph Legend
    direction LR
    stor1[("Storage1")]
    stor2[("Storage2")]
    other["Other Thingy"]
  end
```

### What I want
```text
flowchart LR
  subgraph Legend
    direction LR
    stor1[("Storage1")]
    stor2[("Storage2")]
    other["Other Thingy"]
    stor1-->|invisible|stor2-->|invisible|other
  end
```
```mermaid
flowchart LR
  subgraph Legend
    direction LR
    stor1[("Storage1")]
    stor2[("Storage2")]
    other["Other Thingy"]
    stor1-->|invisible|stor2-->|invisible|other
  end
```

**Comment by jpl-jengelke** [comment]
This is apparently a known issue in Mermaid. See this ticket for some workarounds: https://github.com/mermaid-js/mermaid/issues/815 . 

Unfortunately, the ordering ticket has been closed without a fix.

**Comment by knsv** [comment]
I like the idea with the invisible links! We should do that.

**Comment by knsv** [comment]
@jpl-jengelke Reopened #815, not a simple one to fix though. This issue can help. though.

**Comment by flywire** [comment]
@knsv this is a duplicate of (ie newer than) #2805. Would you mind looking at https://github.com/mermaid-js/mermaid/issues/2805#issuecomment-1148595864?

**Comment by fkohrt** [comment]
This issue has been resolved by #4113 and can be closed.

**Comment by Yokozuna59** [comment]
@fkohrt Thanks!
